1_space_odyssey/js/mannuel-9000/ben-9000/app.js

```diff
@@ -0,0 +1,19 @@
+var isAlive = true; 
```

Comment:
please put your named folder into the root of the JS folder, outside out my name folder.
so the path will look like this:
`js/ben-9000/app.js`

Reply:
we did that to:)